Fabric.Authorization.Domain/Services/GroupService.cs

```diff
@@ -202,5 +202,20 @@ public class GroupService
 
             return await _groupStore.DeleteUserFromGroup(group, user);
         }
+
+        public async Task<IEnumerable<Group>> GetGroups(string name, string type)
+        {
+            if (!string.IsNullOrEmpty(type))
+            {
+                type = type.ToLower();
```

Comment:
We have a `CustomSource` constant in `GroupConstants`. Perhaps we can use that here and and another `DirectorySource` constant to that class for use everywhere?

Reply:
good call, I forgot about that.